File-Formats/XlsIO/Working-with-Template-Markers.md

```diff
@@ -81,7 +81,7 @@ An array of data can be binded to the marker in the template document.
 
 The following screenshot represents the input template which has a template marker.
 
-<img src="Working-with-Template-Markers_images/Template_Markers_in_Excel_Spreadsheet_Templates.jpeg" alt="Bind from Array Example" width="100%" Height="Auto"/>
+<img src="working-with-template-markers_images/file-formats-xlsio-bind-from-array-templale.jpeg" alt="Working with Template Markers in File Formats XLSIO Bind from Array Template" width="100%" Height="Auto"/>
```

Comment:
Here existing image name is Fine. So, revert your changes. Just change the image name to small case. Revert in all places.